src/WilliamsonTests.jl

```diff
@@ -1,5 +1,12 @@
 module WilliamsonTests
 
-# Write your package code here.
+using DocStringExtensions
+
+using SpeedyWeather
+using NCDatasets
+
+include("initial_conditions.jl")
```

Comment:
given you want to have all 7 test cases feel free to sort this into folders if easier